modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -149,5 +149,59 @@ definition {
 		}
 	}
 
+	@description = "LPS-143068 - Verify if is possible to edit DateTime on entries from Custom Object"
+	@priority = 5
+	test CanUpdateEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field as UTC") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186247",
+				objectName = "CustomObject186247",
+				pluralLabelName = "Custom Objects 186247");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186247");
```

Comment:
Same thing here, we shouldn't be publishing the object until we're done setting it up